script.js

```diff
@@ -1,4 +1,285 @@
+var gameMode = "draw cards"; // 4 phases - "draw cards", "game", "choice", "results"
+
+// ===== MAIN FUNCTION ===== //
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  //PHASE 1: DRAW CARDS
+  if (gameMode == "draw cards") {
+    // Create object for each player and deal 2 cards each
+    for (var i = 0; i < numOfPlayers; i += 1) {
+      var card1 = newDeck.pop();
+      var card2 = newDeck.pop();
+      var first2Cards = [card1, card2];
+      allCardsDealt.push(first2Cards);
+    }
+    gameMode = "game";
+    myOutputValue += `Drawing <b>first 2 cards</b> 🎲🎲 for each player...`;
+  }
+  //PHASE 2: GAME
+  else if (gameMode == "game") {
+    // Dealer Turn
+    if (playerTurn == dealer) {
+      myOutputValue += `<b><u>Dealer</b></u><br>`;
+      myOutputValue += `Dealer cards are:`;
+      playersBlackjackValue.push("");
+      var dealerCards = "";
+      var currentTotal = 0;
+      var aceCounter = 0;
+      for (var i = 0; i < allCardsDealt[playerTurn - 1].length; i += 1) {
+        dealerCards += `<br>${allCardsDealt[playerTurn - 1][i].name} of ${
+          allCardsDealt[playerTurn - 1][i].emoji
+        }`;
+        if (allCardsDealt[playerTurn - 1][i].name == "ace" && aceCounter == 0) {
+          currentTotal += 11;
+          aceCounter += 1;
+        } else {
+          currentTotal += allCardsDealt[playerTurn - 1][i].blackjackValue;
+        }
+      }
+      if (currentTotal == 21) {
+        myOutputValue += dealerCards;
+        myOutputValue += `<br><br>Dealer has scored a <b>Blackjack</b> ✨! Press '<b>Submit</b>' to see results...`;
+        playersBlackjackValue[playerTurn - 1] = "Blackjack";
+      } else {
+        while (currentTotal <= 16) {
+          dealerCards = "";
+          allCardsDealt[playerTurn - 1].push(newDeck.pop());
+          var currentTotal = 0;
+          var aceCounter = 0;
+          for (var i = 0; i < allCardsDealt[playerTurn - 1].length; i += 1) {
+            dealerCards += `<br>${allCardsDealt[playerTurn - 1][i].name} of ${
+              allCardsDealt[playerTurn - 1][i].emoji
+            }`;
+            if (
+              allCardsDealt[playerTurn - 1][i].name == "ace" &&
+              aceCounter == 0
+            ) {
+              aceCounter += 1;
+              currentTotal += 11;
+            } else {
+              currentTotal += allCardsDealt[playerTurn - 1][i].blackjackValue;
+            }
+          }
+          if (aceCounter != 0 && currentTotal > 21) {
+            currentTotal -= 10;
+          }
+        }
+        myOutputValue += dealerCards;
+        myOutputValue += `<br><br>Total dealer card value is <b>${currentTotal}</b>`;
+        if (currentTotal > 21) {
+          myOutputValue += ` (That looks like a <b>bust</b>! 💣)`;
+        }
+        myOutputValue += `<br>Press '<b>Submit</b>' to see results...`;
+        if (currentTotal > 21) {
+          playersBlackjackValue[playerTurn - 1] = 0;
```

Comment:
first thought: it can be combined
second thought: due to the sequence of the output statements it cannot, 
Refactoring would require that you rethink how you want to structure the output messages, its need to be such that the common messages are on top and everything concatenated after is unique.
Unique messages where the bulk of the message in the same except for certain inputs